plugins/go.sum

```diff
@@ -0,0 +1,781 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
are we adding a `go mod tidy` check to CI for this new module?

Reply:
I think I can modify hack/tidy-check.sh and use the script to check this new module.